### ISSUE TITLE
Broken on 0.4.0 on Windows: Termios is not on windows

### ISSUE THREAD

**Issue by kaine-bruce-dmt** [olzhasar/pytest-watcher#27]
Hey there, refugee from pytest-watch. 

https://docs.python.org/3/library/termios.html

```
PS C:\Users\Kaine> ptw .
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Scripts\ptw.exe\__main__.py", line 4, in <module>
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Lib\site-packages\pytest_watcher\__init__.py", line 1, in <module>
    from .watcher import VERSION, run
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Lib\site-packages\pytest_watcher\watcher.py", line 10, in <module>
    from . import commands
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Lib\site-packages\pytest_watcher\commands.py", line 8, in <module>
    from .terminal import Terminal
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Lib\site-packages\pytest_watcher\terminal.py", line 6, in <module>
    import termios
ModuleNotFoundError: No module named 'termios'
PS C:\Users\Kaine>
```

**Comment by kaine-bruce-dmt** [comment]
Current workaround is to use 0.3.5:
```
pip install pytest-watcher==0.3.5
```

**Comment by olzhasar** [comment]
Hi @kaine-bruce-dmt ,
Thank you for reporting this!

I have just pushed the fix, but I don't have an access to a Windows machine to test it. Would you be able to help with that?
You will need to Install the version from the master branch:

```sh
pip install git+https://github.com/olzhasar/pytest-watcher.git@master

```

**Comment by kaine-bruce-dmt** [comment]
Looks good, thanks for the swift fix!

Install:
```
PowerShell 7.4.1
PS C:\Users\Kaine> pip freeze
certifi==2023.11.17
distlib==0.3.8
filelock==3.13.1
pipenv==2023.11.17
platformdirs==4.1.0
virtualenv==20.25.0
watchdog==3.0.0
PS C:\Users\Kaine> pip install git+https://github.com/olzhasar/pytest-watcher.git@master
Collecting git+https://github.com/olzhasar/pytest-watcher.git@master
  Cloning https://github.com/olzhasar/pytest-watcher.git (to revision master) to c:\users\kaine\appdata\local\temp\pip-req-build-pjz09rix
  Running command git clone --filter=blob:none --quiet https://github.com/olzhasar/pytest-watcher.git 'C:\Users\Kaine\AppData\Local\Temp\pip-req-build-pjz09rix'
  Resolved https://github.com/olzhasar/pytest-watcher.git to commit 93dec6ceef5dde808679cb33b7c0ab7935960b6d
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: watchdog>=2.0.0 in c:\users\kaine\appdata\local\programs\python\python311\lib\site-packages (from pytest-watcher==0.4.0) (3.0.0)
Building wheels for collected packages: pytest-watcher
  Building wheel for pytest-watcher (pyproject.toml) ... done
  Created wheel for pytest-watcher: filename=pytest_watcher-0.4.0-py3-none-any.whl size=11744 sha256=550f718dd818529040bd4dc1cd51f18ce41c6a911a4221bb209da1eed4fd92b0
  Stored in directory: C:\Users\Kaine\AppData\Local\Temp\pip-ephem-wheel-cache-n666g0hr\wheels\c5\c0\92\506f95f9ab6be681cc38f0fe4e71dc938864447f20143f0aae
Successfully built pytest-watcher
Installing collected packages: pytest-watcher
Successfully installed pytest-watcher-0.4.0

[notice] A new release of pip is available: 23.3.2 -> 24.0
[notice] To update, run: python.exe -m pip install --upgrade pip
```

Running a test:
```
PS C:\ARealRepo> ptw .
pytest-watcher version 0.4.0
Runner command: pytest
Waiting for file changes in C:\ARealRepo
[ptw] .\tests\test_canary.py modified
[ptw] .\tests\test_canary.py modified
=================================================================== test session starts ===================================================================
platform win32 -- Python 3.11.7, pytest-7.4.4, pluggy-1.4.0
rootdir: C:\ARealRepo
plugins: anyio-3.7.1, asyncio-0.23.4
asyncio: mode=Mode.STRICT
collected 1 item

tests\test_canary.py .                                                                                                                               [100%]

==================================================================== 1 passed in 0.01s ====================================================================
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Scripts\ptw.exe\__main__.py", line 7, in <module>
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Lib\site-packages\pytest_watcher\watcher.py", line 72, in run
    main_loop(trigger, config, term)
  File "C:\Users\Kaine\AppData\Local\Programs\Python\Python311\Lib\site-packages\pytest_watcher\watcher.py", line 42, in main_loop
    time.sleep(LOOP_DELAY)
KeyboardInterrupt
PS C:\ARealRepo>
```

**Comment by olzhasar** [comment]
v0.4.1 is out with a fix.

@kaine-bruce-dmt  Thank you for your help!
